lib/AST/Substitution.cpp

```diff
@@ -63,7 +70,8 @@ Substitution Substitution::subst(Module *module,
     // If we have a concrete conformance, we need to substitute the
     // conformance to apply to the new type.
     if (c.isConcrete()) {
-      auto substC = c.getConcrete()->subst(module, substReplacement, subMap);
+      auto substC = c.getConcrete()->subst(module, substReplacement,
```

Comment:
I could be wrong but it appears the module is plumbed through recursively through subst() on conformances and Substitution but it's not actually needed